src/modules/firebase/models/Ordering.ts

```diff
@@ -0,0 +1,11 @@
+export enum Order {
```

Comment:
Same as for collections. Make it a string union.

Also, enums are constants, not models if we will be nitpicky.